apps/omg_db/test/omg_db/recorder_test.exs

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.RecorderTest do
+  @moduledoc """
+  A smoke test of the LevelDB support. The intention here is to **only** test minimally, that the pipes work.
+
+  For more detailed persistence test look for `...PersistenceTest` tests throughout the apps.
+
+  Note the excluded moduletag, this test requires an explicit `--include wrappers`
+  """
+  use ExUnitFixtures
+  use OMG.DB.Case, async: true
+  alias OMG.DB
+  @moduletag :wrappers
```

Comment:
if this test isn't just redundant testing the `wrappers`, which it looks like the case here, this tag isn't needed - let's run these tests always as part of the `mix test` invocation